internal/configtxgen/encoder/encoder_test.go

```diff
@@ -46,7 +46,115 @@ func CreateStandardPolicies() map[string]*genesisconfig.Policy {
 	}
 }
 
+func CreateStandardOrdererPolicies() map[string]*genesisconfig.Policy {
+	return map[string]*genesisconfig.Policy{
+		"Admins": {
+			Type: "ImplicitMeta",
+			Rule: "ANY Admins",
+		},
+		"Readers": {
+			Type: "ImplicitMeta",
+			Rule: "ANY Readers",
+		},
+		"Writers": {
+			Type: "ImplicitMeta",
+			Rule: "ANY Writers",
+		},
+		"BlockValidation": {
+			Type: "ImplicitMeta",
+			Rule: "ANY Admins",
+		},
+	}
+}
+
 var _ = Describe("Encoder", func() {
+	Describe("AddOrdererPolicies", func() {
+		var (
+			cg       *cb.ConfigGroup
+			policies map[string]*genesisconfig.Policy
+		)
+
+		BeforeEach(func() {
+			cg = protoutil.NewConfigGroup()
+			policies = CreateStandardOrdererPolicies()
+		})
+
+		It("adds the policies to the group", func() {
+			err := encoder.AddOrdererPolicies(cg, policies, "Admins")
+			Expect(err).NotTo(HaveOccurred())
+			Expect(len(cg.Policies)).To(Equal(4))
+			expectedImplicitMetaPolicy := &cb.ImplicitMetaPolicy{
+				SubPolicy: "Admins",
+				Rule:      cb.ImplicitMetaPolicy_ANY,
+			}
+			value, err := proto.Marshal(expectedImplicitMetaPolicy)
+			Expect(err).NotTo(HaveOccurred())
+			expectedAdminPolicy := &cb.Policy{
+				Type:  int32(cb.Policy_IMPLICIT_META),
+				Value: value,
+			}
+			Expect(cg.Policies["Admins"].Policy).To(Equal(expectedAdminPolicy))
+
+			expectedImplicitMetaPolicy = &cb.ImplicitMetaPolicy{
+				SubPolicy: "Readers",
+				Rule:      cb.ImplicitMetaPolicy_ANY,
+			}
+			value, err = proto.Marshal(expectedImplicitMetaPolicy)
+			Expect(err).NotTo(HaveOccurred())
+			expectedReadersPolicy := &cb.Policy{
+				Type:  int32(cb.Policy_IMPLICIT_META),
+				Value: value,
+			}
+			Expect(cg.Policies["Readers"].Policy).To(Equal(expectedReadersPolicy))
+
+			expectedImplicitMetaPolicy = &cb.ImplicitMetaPolicy{
+				SubPolicy: "Writers",
+				Rule:      cb.ImplicitMetaPolicy_ANY,
+			}
+			value, err = proto.Marshal(expectedImplicitMetaPolicy)
+			Expect(err).NotTo(HaveOccurred())
+			expectedWritersPolicy := &cb.Policy{
+				Type:  int32(cb.Policy_IMPLICIT_META),
+				Value: value,
+			}
+			Expect(cg.Policies["Writers"].Policy).To(Equal(expectedWritersPolicy))
+
+			expectedImplicitMetaPolicy = &cb.ImplicitMetaPolicy{
+				SubPolicy: "Admins",
+				Rule:      cb.ImplicitMetaPolicy_ANY,
+			}
+			value, err = proto.Marshal(expectedImplicitMetaPolicy)
+			Expect(err).NotTo(HaveOccurred())
+			expectedBlockValidationPolicy := &cb.Policy{
+				Type:  int32(cb.Policy_IMPLICIT_META),
+				Value: value,
+			}
```

Comment:
This looks like a good candidate for "marshal or panic." 

```go
Expect(cg.Policies["BlockValidation"].Policy).To(Equal(&cb.Policy{
    Type: int32(cb.Policy_IMPLICIT_META),
    Value: protoutil.MarshalOrPanic(&cb.ImplicitMetaPolicy{
        SubPolicy: "Admins",
        Rule: cb.ImplicitMetaPolicy_ANY,
     }),
})
```
I also think you can use probably build around CreateStandardPolicies for the first three policies.